projects/1.21/assets/creeperhost-presents-steves-carts/stevescarts/lang/zh_cn.json

```
@@ -0,0 +1,711 @@
+  "block.stevescarts.enhanced_galgadorian_metal": "强化混合晶化金属块",
+  "block.stevescarts.galgadorian_metal": "混合晶化金属块",
+  "block.stevescarts.reinforced_metal": "强化金属块",
+  "block.stevescarts.upgrade_batteries": "升级：电池",
+  "block.stevescarts.upgrade_cart_crane": "升级：吊车",
+  "block.stevescarts.upgrade_cart_deployer": "升级：矿车部署器",
+  "block.stevescarts.upgrade_cart_modifier": "升级：矿车调节器",
+  "block.stevescarts.upgrade_cotwo_friendly": "升级：环保",
+  "block.stevescarts.upgrade_creative_mode": "升级：创造模式",
+  "block.stevescarts.upgrade_entropy": "升级：熵",
+  "block.stevescarts.upgrade_experienced_assembler": "升级：熟练装配工",
+  "block.stevescarts.upgrade_generic_engine": "升级：通用引擎",
+  "block.stevescarts.upgrade_industrial_espionage": "升级：工业间谍",
+  "block.stevescarts.upgrade_manager_bridge": "升级：管理器桥梁",
+  "block.stevescarts.upgrade_module_input": "升级：模块输入",
+  "block.stevescarts.upgrade_module_knowledge": "升级：模块知识",
+  "block.stevescarts.upgrade_new_era": "升级：新纪元",
+  "block.stevescarts.upgrade_power_crystal": "升级：能量水晶",
+  "block.stevescarts.upgrade_production_line": "升级：生产线",
+  "block.stevescarts.upgrade_quick_demolisher": "升级：快速拆卸器",
+  "block.stevescarts.upgrade_redstone_control": "升级：红石控制",
+  "block.stevescarts.upgrade_solar_panel": "升级：太阳能板",
+  "block.stevescarts.upgrade_thermal_engine": "升级：热能引擎",
+  "entity.stevescarts.modular_cart": "模块化矿车",
+  "gui.stevescarts.assembleCart": "装配矿车",
+  "gui.stevescarts.assembleProgress": "进度",
+  "gui.stevescarts.basicAssembleInstruction": "请将一个矿车外壳放到外壳槽里，以开始装配一辆矿车。",
+  "gui.stevescarts.busyAssemblerError": "装配机正在忙碌中！",
+  "gui.stevescarts.cargoManager": "货物",
+  "gui.stevescarts.cartAreaAll": "所有槽位",
+  "gui.stevescarts.cartAreaArrows": "弓箭",
+  "gui.stevescarts.cartAreaBridge": "桥材料",
+  "gui.stevescarts.cartAreaBuckets": "桶（给挤奶工用的）",
+  "gui.stevescarts.cartAreaCakes": "蛋糕",
+  "gui.stevescarts.cartAreaEngine": "引擎",
+  "gui.stevescarts.cartAreaExplosives": "爆炸物",
+  "gui.stevescarts.cartAreaFertilizer": "施肥器",
+  "gui.stevescarts.cartAreaFirework": "烟花",
+  "gui.stevescarts.cartAreaRailer": "铁轨",
+  "gui.stevescarts.cartAreaSaplings": "树苗",
+  "gui.stevescarts.cartAreaSeeds": "种子",
+  "gui.stevescarts.cartAreaStorage": "存储槽",
+  "gui.stevescarts.cartAreaTorches": "火把",
+  "gui.stevescarts.cartAssembler": "矿车装配机",
+  "gui.stevescarts.changeSide": "改变方向",
+  "gui.stevescarts.changeSlotLayout": "改变槽位的布局",
+  "gui.stevescarts.changeTankLayout": "改变储罐布局",
+  "gui.stevescarts.changeTransferCartArea": "改变矿车货物",
+  "gui.stevescarts.changeTransferDirection": "改变转移方向",
+  "gui.stevescarts.changeTransferSize": "改变转移大小",
+  "gui.stevescarts.changeTurnBack": "改变返回设置",
+  "gui.stevescarts.complexityCap": "复杂度上限",
+  "gui.stevescarts.continueForward": "转移后继续前进",
+  "gui.stevescarts.currentSetting": "当前",
+  "gui.stevescarts.currentSide": "当前方向",
+  "gui.stevescarts.departureBayError": "成品槽里有一辆矿车。",
+  "gui.stevescarts.directionFromCart": "从矿车",
+  "gui.stevescarts.directionToCart": "至矿车",
```

```suggestion
  "gui.stevescarts.directionFromCart": "取离矿车",
  "gui.stevescarts.directionToCart": "送入矿车",
```
仅建议，如改下方`distributor`部分同文键也改